src/me/mrCookieSlime/Slimefun/listeners/BackpackListener.java

```diff
@@ -65,7 +66,7 @@ public void onClick(InventoryClickEvent e) {
 			else {
 				SlimefunItem sfItem = SlimefunItem.getByItem(e.getCurrentItem());
 				if (SlimefunManager.isItemSimiliar(item, SlimefunItem.getItem("COOLER"), false)) {
-					if (e.getCurrentItem() == null);
+					if (e.getCurrentItem().getType() == Material.AIR);
 					else if (sfItem == null) e.setCancelled(true);
```

Comment:
sfItem == null 
can be removed, instanceof takes care of that.
I probably didn't know about that, back when I wrote this code.